packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -7,6 +7,7 @@ import type { SanitizedAdminUser } from '@strapi/admin/strapi-admin';
 export interface Release extends Entity {
   name: string;
   releasedAt: string;
+  scheduledAt: string;
```

Comment:
If you can create a release without scheduling, should this be optional?

Reply:
I tried first with `scheduledAt: string | null` because we return the scheduledAt as null when is not scheduled (is the same for releasedAt)

But I don't know why when you do that TS said the scheduledAt type is still only `string` 🤔

Reply:
I believe ,when not set, teh value will be undefined, so probably `scheduledAt?: string` should work (?)

Reply:
I changed it to `string | null` which I think represents better the case, because on the response we are still sending scheduledAt and releasedAt as null. Let me know what you think

Reply:
sounds good :)